src/PhpSpreadsheet/Style/Font.php

```diff
@@ -255,9 +255,16 @@ public function getSize()
      */
     public function setSize($pValue)
```

Comment:
The docblock needs modifying to `mixed` to keep phpstan happy, and it looks like phpcs is complaining about trailing white space in the comment lines.
If you can just make those changes, then the PR is good to merge